concordia/templates/emails/welcome_email_subject.txt

```diff
@@ -0,0 +1 @@
+Welcome to By The People at crowd.loc.gov
```

Comment:
@elainekamlley @lalgee Welcome email subject - currently disabled / behind a feature flag. The user would receive this email when the account goes from inactive to active - whether that's through password reset or the regular confirm email process.

Reply:
I don't think we need the url in the subject line. Welcome to By the People is good.